contracts/token/ERC721/ItemNFT.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-FileCopyrightText: 2023 ISKRA Pte. Ltd.
+// SPDX-License-Identifier: MIT
+// @author iskra.world dev team
+
+pragma solidity ^0.8.2;
+
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/access/Ownable2Step.sol";
+
+// ItemNFT features
+//  - ERC721 basic functions
+contract ItemNFT is ERC721, Ownable2Step {
+    string internal baseURI = "";
```

Comment:
```suggestion
    string internal _baseURI = "";
```

Reply:
`_baseURI` is already used for the `_baseURI()` internal function, so I renamed it to `mBaseURI`.